integrations/cdi/jpa-cdi/src/test/java/io/helidon/integrations/cdi/jpa/chirp/TestJpaTransactionScopedSynchronizedEntityManager.java

```diff
@@ -33,6 +33,7 @@
 import jakarta.inject.Inject;
 import jakarta.inject.Named;
 import jakarta.persistence.EntityManager;
+import jakarta.persistence.OptimisticLockException;
```

Comment:
Oops; probably need to remove this

Reply:
Resolved in https://github.com/helidon-io/helidon/pull/10904/commits/15ab5cd6e1d9b6b7a012616bc3f6b947d77059f8